### ISSUE TITLE
PICO-8 SSPR() function?

### ISSUE THREAD

**Issue by StinkerB06** [nesbox/TIC-80#952]
[PICO-8](https://www.lexaloffle.com/pico-8.php) has a [`SSPR()` function](https://pico-8.fandom.com/wiki/Sspr), which is useful to draw scaled textured rectangles.

I'd love TIC-80 to have a similar function. Sure, we can make our own `SSPR()` replica in TIC-80 Lua, perhaps with some `pix()` or `textri()` drawing. The only problem with the former is that it's really CPU-intensive to run on web browsers, mobile devices, or low-end PC/laptops at full 60 FPS. And the most annoying part of this is that it causes audio lag.

**Comment by nesbox** [comment]
Hmm, I don't see the reason to add `sspr()` when you could to it just in two 'textri()' calls.

**Comment by StinkerB06** [comment]
The `textri()` method isn't really good at all:
![textri() texture warp](https://user-images.githubusercontent.com/30059831/67682723-3b0cbf80-f94d-11e9-945c-741b6648fea0.gif)

Look at this! Notice the jittery lines on the lower-right `textri()`, and the occasional "glitchy" pixels.

```
W,H=16,16

function TIC()

	if btn(0) then H=H-1 end
	if btn(1) then H=H+1 end
	if btn(2) then W=W-1 end
	if btn(3) then W=W+1 end
	
	cls()
	textri(0,0,W,0,0,H,8,0,24,0,8,16,false,14)
	textri(W,0,0,H,W,H,24,0,8,16,24,16,false,14)
	
end
```
Please correct this code if needed!

**Comment by nesbox** [comment]
yes, doesn't look perfect, anyway, this is a problem of `textri()` function

**Comment by nesbox** [comment]
let's fix it as part of #1059

**Comment by RobLoach** [comment]
Create a new issue. This one is about creating a new sspr() function rather than improving the existing one.
